### PR TITLE
Fixes #434, Improves front-end for advanced search

### LINKED ISSUE #434: Improve front-end for advanced search page
**Actual Behaviour**

Advanced search page is having : 
- Input box with curves at the corner.
- Advanced search button.
- No padding between advanced search button and footer.

**Expected Behaviour**

- Input box should be sharp at corners.
- Improve the advanced search button following the market leader.
- Give some padding between advanced search button and footer so that it looks cleaner.
- Align the advanced search button towards right.

**Screenshots of the issue**

Advanced search button (Google)
screenshot from 2017-06-07 04-01-28

Advanced search button (Susper)
screenshot from 2017-06-07 04-01-41

Alignment and padding (Google)
screenshot from 2017-06-07 04-01-50

Alignment and padding (Susper)
screenshot from 2017-06-07 04-01-59

Input box (Google)
screenshot from 2017-06-07 04-02-16

Input box (Susper)
screenshot from 2017-06-07 04-02-25

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixes issue #434 

Changes: Improves front end for advanced search as mentioned in #434 in the following ways:

1. Changes Search button css

2. Changes Search button positoning

3. Changes input boxes rounded corners

4. Right padding around Advanced Search button(Followed market leader)

Demo Link: [Here](https://marauderer97.github.io/susper.com/)

Screenshots for the change: 

image
image
image

### CODE REVIEW

**Comment by Marauderer97** [comment]
@harshit98 @nikhilrayaprolu @isuruAb @sandeepm96 Please review :smiley:

**Comment by harshit98** [comment]
@Marauderer97 Kindly look into the travis errors. :)

**Comment by Marauderer97** [comment]
@sandeepm96 It's not travis, codacy is failing because I used webkit-linear :worried: Trying to find a solution.

File: src/app/newadvancedsearch/newadvancedsearch.component.css [hunk 1]
```diff
@@ -3,10 +3,48 @@
...
+  border: 1px solid #3079ed;
+  border-radius: 2px;
+}
+
+.btn-text {
+color: white;
+  font-size: 11px;
+  text-align: center;
+font-weight: bold;
+}
```
Comment (harshit98) [inline src/app/newadvancedsearch/newadvancedsearch.component.css]: Indent them. :)

**Pull request merged**